### PR TITLE
HPCC-21702 New Zap options added to ECL Watch doc

### PR DESCRIPTION
**Pull request by g-pan**
Signed-off-by: G-Pan <greg.panagiotatos@lexisnexis.com>

@JamesDeFabia please review
@kunalaswani FYI, optionally review

View compiled PDF at http://10.240.32.243/view/Docs%207.0/job/AnotherDocBuild7

## Type of change:
- [x] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [x] My change requires a change to the documentation.
  - [x] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [ ] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [ ] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Testing:

View Successful Build TEST at http://10.240.32.243/view/Docs%207.0/job/AnotherDocBuild7

### CODE REVIEW

File: docs/EN_US/ECLWatch/ECLWa_mods/ECLWatchSrc.xml [hunk 1]
```diff
@@ -573,11 +573,23 @@
...
+          are slave logs, the option to include them is available. You must
+          check that field when available to include the slave logs. </para>
+
+          <para>The Send Email field is only available if email is configured
+          for the ESP service in the configuration manager. If available,
+          check the Send Email box to email the ZAP report. Only an
+          Administrator can configure the email. The (To) Email Address is
+          also set by the Administrator and can only be changed in the
+          configuration. The (From) Email Address can be set in the
+          configuration manager, but can be changed if desired. The Email
```
Comment (JamesDeFabia) [inline docs/EN_US/ECLWatch/ECLWa_mods/ECLWatchSrc.xml]: Configuration Manager (s/b title case)

**Review by JamesDeFabia — CHANGES_REQUESTED** [review]
We should discuss what to do with sensitive info BEFORE the step that says "Press the Apply button." 

Something along the lines of:
If your ZAP package might contain sensitive information, use the utility to create a package, then sanitize it, and send manually. 
If it doesn't then you can use the email feature to send automatically.

**Comment by JamesDeFabia** [comment]
@g-pan back to you

**Comment by g-pan** [comment]
@JamesDeFabia Revised and Rebased. Please Review.

**Comment by g-pan** [comment]
@JamesDeFabia Revised and Rebased.

File: docs/EN_US/ECLWatch/ECLWa_mods/ECLWatchSrc.xml [hunk 2]
```diff
@@ -573,19 +573,38 @@
...
+          for the ESP service in the Configuration Manager. If available,
+          check the Send Email box to email the ZAP report. Only an
+          Administrator can configure the email. The (To) Email Address is
+          also set by the Administrator and can only be changed in the
+          configuration. The (From) Email Address can be set in the
+          Configuration Manager, but can be changed if desired. The Email
+          Subject is required, but the Email Body is optional.</para>
+
+          <para>Press the <emphasis role="bold">Apply </emphasis>button when
+          all the dialog fields are filled. At that point if you checked the
```
Comment (JamesDeFabia) [inline docs/EN_US/ECLWatch/ECLWa_mods/ECLWatchSrc.xml]: ...all the dialog fields are filled. Should be ...all the dialog fields are completed. (or filled in.)

**Review by JamesDeFabia — CHANGES_REQUESTED** [review]
only one minor comment inline

**Review by JamesDeFabia — APPROVED** [review]
Ok to merge

**Pull request merged**